server/src/test/java/org/elasticsearch/TransportVersionTests.java

```diff
@@ -167,10 +167,14 @@ public void testVersionConstantPresent() {
         }
     }
 
-    public void testCURRENTIsLatest() {
+    public void testCurrentIsLatest() {
         assertThat(Collections.max(TransportVersion.getAllVersions()), is(TransportVersion.CURRENT));
     }
 
+    public void testCurrentIncrementGap() {
+        assertThat(TransportVersion.CURRENT.id() % 100, equalTo(0));
```

Comment:
It's worth adding a comment here so devs know what they should be doing when the test fails